### PR TITLE
[13.0][MIG] stock_picking_package_preparation: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migraton to 13.0

Locked by:
- [x] https://github.com/OCA/stock-logistics-workflow/pull/885
- [x] https://github.com/OCA/stock-logistics-workflow/pull/884

Please @pedrobaeza and @CarlosRoca13  can you review it?

@Tecnativa TT30790

### CODE REVIEW

File: stock_picking_package_preparation/views/stock_picking_package_preparation_view.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
         <field name="name">stock.picking.package.preparation.tree</field>
         <field name="model">stock.picking.package.preparation</field>
         <field name="arch" type="xml">
-            <tree colors="grey:state == 'cancel';" string="Package Preparations">
+            <tree decoration-info="state == 'cancel'" string="Package Preparations">
```
Comment (CarlosRoca13) [inline stock_picking_package_preparation/views/stock_picking_package_preparation_view.xml]: I think that decoration-muted could be a better option. In previous version the module use color grey, so we can set the same color.
```suggestion
            <tree decoration-muted="state == 'cancel'" string="Package Preparations">
```

**Review by CarlosRoca13 — APPROVED** [review]
Just a minor comment

**Comment by sebalix** [comment]
<s>The tests failing in some modules is probably due to this commit on Odoo: https://github.com/odoo/odoo/commit/591a726f4901b863f7a13581942e7e6b352c1728
We fix it on `shopfloor` here: https://github.com/OCA/wms/pull/275 but it's maybe not the cleanest way...</s>

Odoo fixes their code: https://github.com/odoo/odoo/pull/75019 , I restarted the build to check

**Review by sebalix — APPROVED** [review]

**Pull request merged**